CONTRIBUTING.md

```diff
@@ -88,6 +88,15 @@ Then to check if Hugepages is enabled:
 python3 infra/machine_setup/scripts/setup_hugepages.py check
 ```
 
+### Software dependencies for codebase contributions
```

Comment:
Moved this step from INSTALLING.md to CONTRIBUTING.md as it is specific to contributing.

Reply:
Sure, works for me.